src/test/java/org/assertj/core/api/string_/StringAssert_isEqualToNormalizingUnicode_Test.java

```diff
@@ -1,41 +0,0 @@
-package org.assertj.core.api.string_;
-
-import org.assertj.core.api.StringAssert;
-import org.assertj.core.api.StringAssertBaseTest;
-import org.junit.Test;
-
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.util.AssertionsUtil.expectAssertionError;
-import static org.mockito.Mockito.verify;
-
-public class StringAssert_isEqualToNormalizingUnicode_Test extends StringAssertBaseTest {
```

Comment:
keep this one, simply remove `should_pass_equals_normalized_String_when_compared_with_normalized_String` and `should_fail_if_denormalized_String_is_compared_with_normalized_String`

Reply:
I believe this code is not currently available in the current commits 🤔